docs/team/fungusta.md

```diff
@@ -19,16 +20,17 @@ Given below are my contributions to the project.
 * **Code contributed**: [RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=fungusta&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2022-09-16&tabOpen=true&tabType=authorship&tabAuthor=fungusta&tabRepo=AY2223S1-CS2103T-T10-2%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=functional-code~test-code&authorshipIsBinaryFileTypeChecked=false&authorshipIsIgnoredFilesChecked=false)
 
 * **Project management**:
-    * [to be added soon]
+    * Set up GitHub Issues and Milestones for tasks to be completed before certain deadlines.
+    * Contributed in reviewing Pull Requests made by other members of the team.
 
 * **Enhancements to existing features**:
     * Contact Finding by Address
-      * Finds and lists out all people in FABook that address contains any of the keywords input by user
+      * Finds and lists out all people in FABook that address contains any of the keywords input by user.
+      * Improved polymorphism of `Command` class by adding an abstract child `FindCommand` for find commands.
```

Comment:
I think this can be a main point instead of sub-point

Reply:
I think because it was done when enhancing contact finding, should be ok? what you think?

Reply:
Either way is fine 👍